Events/LocationEvent.cs

```diff
@@ -59,6 +59,8 @@ static LocationEvent()
 
         public string faction { get; private set; }
 
+        public string factionstate => (factionState ?? FactionState.None).localizedName;
```

Comment:
Not too keen on the subtle case distinction between `factionstate` and `factionState`. I suppose it's for Cottle?